tests/e2e/policy_driven_vol_allocation.go

```diff
@@ -50,12 +50,18 @@ var pc *pbm.Client
 var spareSpace int64 = 200
 
 var _ = ginkgo.Describe("[vol-allocation] Policy driven volume space allocation tests", func() {
-
 	f := framework.NewDefaultFramework("e2e-spbm-policy")
+	if supervisorCluster {
+		framework.TestContext.DeleteNamespace = false
```

Comment:
so do we collect events from the SVC namespace in case of test failure?

Reply:
Yes, with these changes we are able to collect events from correct SVC namespace.